pkpdapp/pkpdapp/settings.py

```diff
@@ -37,6 +37,8 @@
 INSTALLED_APPS = [
     # standard Django apps
     'django.contrib.admin',
+    'django.contrib.admindocs',
+    # 'django.contrib.admindocs.middleware.XViewMiddleware',
```

Comment:
Again probably naive but how come this line is commented out?

Reply:
no idea, I'll remove it